src/components/App/App.js

```diff
@@ -229,66 +211,40 @@ function App() {
             }
           />
         </Routes>
-        <PopupWithForm
-          handleLogin={handleLogin}
-          loggedIn={loggedIn}
-          onOpen={openRegisterPopup}
-          submitHandler={handleLogin}
-          isOpen={isSignInPopupOpen}
-          onClose={closeAllPopups}
-          name="sign-in"
-          title="Sign in"
-          buttonText="Sign in"
-          linkText="Sign up"
-        >
-          <PopupInput
-            title="Email"
-            id="input_type_email"
-            type="email"
-            placeholder="Enter email"
-            name="Email"
-          />
-          <PopupInput
-            title="Password"
-            id="input_type_password"
-            type="password"
-            placeholder="Enter password"
-            name="Password"
-          />
-        </PopupWithForm>
-        <PopupWithForm
-          onSignInPopupClick={handleFormClick}
-          submitHandler={handleRegister}
-          isOpen={isSignUpPopupOpen}
-          onOpen={openLoginPopup}
-          onClose={closeAllPopups}
-          name="sign-up"
-          title="Sign up"
-          buttonText="Sign up"
-          linkText="Sign in"
-        >
-          <PopupInput
-            title="Email"
-            id="input_type_email"
-            type="email"
-            placeholder="Enter email"
-            name="Email"
-          />
-          <PopupInput
-            title="Password"
-            id="input_type_password"
-            type="password"
-            placeholder="Enter password"
-            name="Password"
-          />
-          <PopupInput
-            title="Username"
-            id="input_type_username"
-            type="text"
-            placeholder="Enter your username"
-            name="Username"
-          />
-        </PopupWithForm>
+        <Login>
+          <PopupWithForm
```

Comment:
```jsx

<PopupWithForm
            isOpen={isSignInPopupOpen}
            onClose={closeAllPopups}
            onOpen={openRegisterPopup}
            name="sign-in"
            title="Sign in"
            linkText="Sign up"
            buttonText="Sign in"
            submitHandler={handleLogin}
            signError={signError}
          >
            <PopupInput title="Email" type="email" name="email" />
            <PopupInput title="Password" type="password" name="password" />
          </PopupWithForm>
        </Login>
        <Register>
          <PopupWithForm
            onSignInPopupClick={handleFormClick}
            submitHandler={handleRegister}
            isOpen={isSignUpPopupOpen}
            onOpen={openLoginPopup}
            onClose={closeAllPopups}
            name="sign-up"
            title="Sign up"
            buttonText="Sign up"
            linkText="Sign in"
            signError={signError}
          >
            <PopupInput title="Email" type="email" name="email" />
            <PopupInput title="Password" type="password" name="password" />
            <PopupInput title="Username" type="text" name="username" />
          </PopupWithForm>

```


no need to put this markup in `App`. You have `Login` component for that (you should hide the whole markup inside `Login`  and use here only this



```jx

        <Login
            isOpen={isSignInPopupOpen}
            onClose={closeAllPopups}
            onOpen={openRegisterPopup}
            submitHandler={handleLogin}
            signError={signError}
        />


```


and the same with `Register`